### PR TITLE
Added transpiled version

### PR DESCRIPTION
**Pull request by ksnll**
This will transpile the lib in a dist/ folder
The size will go up from 5.4K to 8.6K (+60%)

#2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # qsomeness
-A zero-dependencies tiny < 5kb tool to work with url querystrings. Works both on server and client. It automatically encode/decode params.
+A zero-dependencies tiny ~ 8kb tool to work with url querystrings. Works both on server and client. It automatically encode/decode params.
```
Comment (stecb) [inline README.md]: We can also minify it after babelising.

Comment (stecb) [reply README.md]: Or, write the minified/gzipped kb in parenthesis

**Comment by stecb** [comment]
The growth of size is fine, as the transpiled version contains transpiled helpers.

**Comment by ksnll** [comment]
💯

**Pull request merged**